helm/config/images-arm.env

```diff
@@ -0,0 +1,2 @@
+RAY_IMAGE_IMG_REPO=rayproject/ray
```

Comment:
I also see that these are ARM values, so that would require us to publish an ARM version first I see

Reply:
this is just for temporary thing for my local to work properly. Otherwise I couldn't make my local work without it. Somehow trying to make it generic while not making a significant changes. By default `./config/images.env` is used, but if the platform is on an ARM chip, then it is overridden with this value. If I'd change it latest-cpu then at least my local doesn't work and I'd struggle testing.

Reply:
All right, yeah it's fine to keep that in for testing purposes